tests/test_determinism.py

```diff
@@ -70,8 +70,8 @@ def test_deterministic_energies():
             bps.append(bp.to_gpu(precision=precision).bound_impl)  # get the bound implementation
             ubps.append(bp.potential.to_gpu(precision).unbound_impl)
 
-        for barostat in [None, baro.impl(bps)]:
-            ctxt = custom_ops.Context(x0, v0, complex_box, intg.impl(), bps, barostat=barostat)
+        for movers in [None, [baro.impl(bps)]]:
```

Comment:
Should we add a determinism test for:

`[tibdem]`
`[baro.impl(bps), tibdem]`?

Reply:
Done in [a3b29b5](https://github.com/proteneer/timemachine/pull/1206/commits/a3b29b595f246efdba69a737f8aeb2a617ac5e7f) and [1e27f26](https://github.com/proteneer/timemachine/pull/1206/commits/1e27f2664a98992f887020bbdacbcf82af11c761)